front-end/src/store/transcriptUploadSlice.js

```diff
@@ -3,15 +3,23 @@ import { createSlice } from '@reduxjs/toolkit'
 export const transcriptUploadSlice = createSlice({
   name: 'transcriptUpload',
   initialState: {
-    isUploaded: false
+    isUploaded: false,
+    uploadedProjectIds : []
   },
   reducers: {
     setTranscriptUploadStatus: (state, action) => {
       state.isUploaded = action.payload
+      if (!(action.payload)){
+        // reset projectIds if upload was not successful
+        state.uploadedProjectIds = [];
```

Comment:
I was thinking uploadedProjectIds might be good as a separate variable that doesn't get reset if upload wasn't successful. It could actually be great for cases when the user uploads multiple transcripts in the future, or even if the user uploads two transcripts with different project ids but want to be able to switch between the two in the same session. So maybe reset uploadedProjectIds when user closes the browser? Just an idea

Reply:
Yeah that's a good idea. But we still need one variable that represents the project ids that has been uploaded in this use case. Maybe we can have both if I add another variable.

Reply:
Ah I see that makes more sense. I was confused about the difference between keeping track of uploaded vs analyzed transcripts.

Reply:
yeah It's kind of confusing but I tried renaming the displayed project id to make it more clear. It's just important for us to keep the use cases separated for the sake of CA.